sacn/__init__.py

```diff
@@ -0,0 +1,8 @@
+# re-export the classes available to consumers of this library
```

Comment:
Same question for this dependency.
I guess we should setup a way to track these embedded dependencies, so we can eventually update them when needed.

Reply:
I added dependencies tracking file, as well as changelog and development files.